muikku-core-plugins/src/main/resources/fi/otavanopisto/muikku/plugins/transcriptofrecords/TranscriptofRecordsJsPlugin_en.properties

```diff
@@ -41,7 +41,10 @@ plugin.records.summary.notifications.title = Notifications
 plugin.records.yo.title = Your matricular examinaition
```

Comment:
Not part of this PR but should be changed. Suggested change: Matriculation examination